schema-loader/src/main/java/com/scalar/db/schemaloader/core/SchemaOperatorFactory.java

```diff
@@ -1,19 +1,93 @@
 package com.scalar.db.schemaloader.core;
 
+import com.scalar.db.api.DistributedStorageAdmin;
 import com.scalar.db.config.DatabaseConfig;
+import com.scalar.db.transaction.consensuscommit.ConsensusCommitAdmin;
 import java.io.FileInputStream;
 import java.io.IOException;
 import java.nio.file.Path;
 import java.util.Properties;
 
 public class SchemaOperatorFactory {
-  public static SchemaOperator getSchemaOperator(Properties properties) {
+  /**
+   * Creates a SchemaOperator instance.
+   *
+   * @param properties the Scalar DB properties configuration
+   * @param separateCoordinatorTable separate creating/deleting coordinator table or not. If this
+   *     param is `true` then for example when creating a schema which contains a transactional
+   *     table, the coordinator will be created as well.
+   */
+  public static SchemaOperator getSchemaOperator(
+      Properties properties, boolean separateCoordinatorTable) {
     DatabaseConfig databaseConfig = new DatabaseConfig(properties);
-    return new SchemaOperator(databaseConfig, true);
+    return new SchemaOperator(databaseConfig, separateCoordinatorTable);
   }
 
-  public static SchemaOperator getSchemaOperator(Path configPath) throws IOException {
-    DatabaseConfig dbConfig = new DatabaseConfig(new FileInputStream(configPath.toString()));
-    return new SchemaOperator(dbConfig, false);
+  /**
+   * Creates a SchemaOperator instance.
+   *
+   * @param configPath the file path to Scalar DB configuration file
+   * @param separateCoordinatorTable separate creating/deleting coordinator table or not. If this
+   *     param is `true` then for example when creating a schema which contains a transactional
+   *     table, the coordinator will be created as well.
+   */
+  public static SchemaOperator getSchemaOperator(
```

Comment:
Looks like this method is never used.

Reply:
Can we just leave it out? I supposed that `Operator` and its operations including creating/deleting tables, coordinator table can be directly used by getting a `Operator` instance bay various ways?

Reply:
Since `SchemaOperator` is an internally used class (we don't assume users use it), I thought we could add the method when we need it. If we have unnecessary variations, that increases the test cost.

If we assume that users use the method, that's fine to have multiple variations to cover many use cases. But if not, we should have the minimum necessary.

What do you think?

Reply:
Thank you. I got the point. I removed them in [commit](https://github.com/scalar-labs/scalardb/pull/366/commits/219473b5710f1f7154ae5075eaa1531d48b854d6)